x/metadata/spec/04_authz.md

```diff
@@ -0,0 +1,72 @@
+# Metadata Authz
+
+The `authz` implementation in the `metadata` module checks for granted permission in cases when there are missing signatures.
+
+A `GenericAuthorization` should be used using the message type URLs now documented in [03_messages.md](03_messages.md).
+
+## Code
```

Comment:
From the root of the repo, please run this command:
```
> scripts/update-toc.sh x/metadata/spec/04_authz.md
```

It will add a table of contents to this file.

Alternatively, you can add `<!-- TOC -->` on it's own line above the first `##` line, then run `make update-tocs`. The, `make update-tocs` command won't add a table of contents to a file that doesn't have one, it'll only update files with existing ones.